nr/data/settings.js

```diff
@@ -227,7 +227,7 @@ module.exports = {
             // debug - record information which is more verbose than info + info + warn + error + fatal errors
             // trace - record very detailed logging + debug + info + warn + error + fatal errors
             // off - turn off all logging (doesn't affect metrics or audit)
-            level: "info",
+            level: "trace",
```

Comment:
I believe convention is to leave the master (i.e. production) settings for things like logs at `info` level. Configuring environments for stuff like this might be a good feature down the road too.